### ISSUE TITLE
post-renderer

### ISSUE THREAD

**Issue by revolunet** [SocialGouv/kube-workflow#55]
Looks like `.kube-workflow/post-renderer` is not executed.

The script in https://github.com/SocialGouv/kube-workflow/blob/master/packages/workflow/bin/post-renderer tries to trigger one but looks like the repo `post-renderer` script is not included is the `KW_BUILD_PATH`

test looks broken too https://github.com/SocialGouv/kube-workflow/tree/master/packages/workflow/tests/samples/post-renderer

**Comment by devthejo** [comment]
yep, je l'avais constaté mais dans kontinuous ça fonctionne, du coup pas certain que ça vaille le coup de débugger ça, tu peux utiliser les patches js si besoin, eg: https://github.com/SocialGouv/domifa/blob/3ce8782e4dc5d626be1371e670a4045664ba730d/.kube-workflow/patches/index.js
